app/javascript/packages/verify-flow/verify-flow-step-indicator.tsx

```diff
@@ -62,16 +59,46 @@ export function getStepStatus(index, currentStepIndex): StepStatus {
   return StepStatus.INCOMPLETE;
 }
 
+/**
+ * Given contextual details of the current flow path, returns explicit statuses which should be used
+ * at particular steps.
+ *
+ * @param details Flow details
+ *
+ * @return Step status overrides.
+ */
+function getStatusOverrides({
+  addressVerificationMethod,
+}: {
+  addressVerificationMethod: AddressVerificationMethod;
+}) {
+  const statuses: Partial<Record<VerifyFlowStepIndicatorStep, StepStatus>> = {};
+
+  if (addressVerificationMethod === 'gpo') {
+    statuses.verify_phone_or_address = StepStatus.PENDING;
+  }
+
+  return statuses;
+}
+
 function VerifyFlowStepIndicator({ currentStep }: VerifyFlowStepIndicatorProps) {
   const currentStepIndex = STEP_INDICATOR_STEPS.indexOf(FLOW_STEP_STEP_MAPPING[currentStep]);
+  const [addressVerificationMethod] = useContext(AddressVerificationMethodContext);
+  const statusOverrides = getStatusOverrides({ addressVerificationMethod });
+
+  // i18n-tasks-use t('step_indicator.flows.idv.getting_started')
```

Comment:
I had to move `i18n-tasks-use` comments into the component definition from the top of the file. I'm a bit confused why these weren't being picked up, or why it was suddenly breaking now and not previously. My hunch is that it's some combination of (a) Webpack tree-shaking to eliminate top-level content aside from the specific imported content and (b) internal Webpack chunk optimization kicking in only after the file reached a particular size. In any case, it's probably better to have the comments close to where they're actually used in code, but this will be something to keep an eye out in the future. Fortunately, the specs did a decent job of highlighting the issue.